lib/ControlledVocab/ControlledVocab.js

```diff
@@ -485,6 +487,7 @@ class ControlledVocab extends React.Component {
               // TODO: not sure why we need this OR if there are no groups
               // Seems to load this once before the groups data from the manifest
               // is pulled in. This still causes a JS warning, but not an error
+              canCreate={this.props.canCreate}
               contentData={rows}
```

Comment:
@vashjs, this new prop, `canCreate`, separates a comment block (`TODO...`) from the prop it describes (`contentData`). 

In fact, the comment refers to [an old formulation of `contentData`](https://github.com/folio-org/stripes-smart-components/commit/d08eac47509670778648d68fba7e8e6e4406f5fd#diff-0748e4d1503798efcf342204a4077ea037fc95a88d0408661cee6a2e947b3b07R89) so it should just be removed, but please take note of this situation. If the comment were still relevant then this separation would be unfortunate.

Reply:
@zburke thanks for catching this.